doc/ref_model/chapters/chapter03.rst

```diff
@@ -357,10 +357,94 @@ The hardware infrastructure manager needs to support the following functional as
+
+-	**Equipment**: The Redfish schema contains a physical model and a functional model. The physical model expresses the chassis and containers and interconnects between them and environmental control functions.  The functional model expresses the logical aspects and includes configuration and  firmware status.   Both the physical and functional model have their own health/fault status information
+
+-	**Resource Allocation and Composition**: The Redfish schema has a composition model through which a client can compose a logical resource by allocating physical resources. `Redfish Composition Whitepaper <https://www.dmtf.org/sites/default/files/standards/documents/DSP2050_1.2.0_1.pdf>`__
+
+-	**Underlay Network Resources Manager**: The Redfish schema has models for fabrics, switches and SmartNICs.
+
+-	**Monitoring**: The Redfish schema contains event model for the client to receive hardware events and telemetry model for collecting information across the entire model (physical and functional) `Redfish Telemetry Whitepaper <https://www.dmtf.org/sites/default/files/standards/documents/DSP2051_1.0.0.pdf>`__
+
+-	**Additional Management Functions**: The Redfish schema has models for access management and identity management. Redfish has published work-in-progress extension for support policy management and OS/VM management `Policy Model Proposal, Redfish for Operating Systems, Applications and Containers <https://www.dmtf.org/documents/redfish-spmf/redfish-operating-systems-applications-and-containers-05wip>`__
+
+The ODIM implementation supports the account, aggregation, composition, events, fabric, license, managers, session, systems, task, 
+telemetry and update models.
+
+Open Distributed Infrastructure Management (ODIM™) is a Linux Foundation Networking open source project.  The project has developed a modular, open framework for simplified management and orchestration of distributed, multi-vendor, physical infrastructure.  More information about ODIM is available at `odim.io <https://odim.io>`__.
+
+ODIM comprises the following two key components:
+
+1.	The resource aggregation function
+
+The resource aggregation function is the single point of contact between the northbound clients and the southbound infrastructure. Its primary function is to build and maintain a central resource inventory. It exposes Redfish-compliant APIs to allow northbound infrastructure management systems to:
+
+-	Get a unified view of the southbound compute, local storage, and Ethernet switch fabrics available in the resource inventory
+
+-	Gather crucial configuration information about southbound resources
+
+-	Manipulate groups of resources (“aggregates”) in a single action
+
+-	Listen to similar events from multiple southbound resources
+
+2.	PLugin layer
+
+This consists of one or more plugins, which abstract, translate, and expose southbound resource information to the resource aggregator through RESTful APIs.
+	
```

Comment:
It looks like this line makes the build fail (./chapters/chapter03.rst:423: D003 Tabulation used for indentation)

https://github.com/anuket-project/anuket-specifications/actions/runs/4342322387/jobs/7583535428

Reply:
Is there a guide covering these things ? I am new to Anuket doc reqs.

Reply:
Sorry, we do not have a documentation guide, this could be fixed by simply providing a correct rst file. 
You can always build your doc locally with `tox -e docs` in the RM folder.

Reply:
Fixed